modules/tensor_mechanics/src/kernels/SolidMechInertialForceTensors.C

```diff
@@ -0,0 +1,53 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+#include "SolidMechInertialForceTensors.h"
```

Comment:
Thanks for the comments. I will edit the files and resubmit.

On Fri, Aug 14, 2015 at 2:38 PM, Daniel Schwen <notifications@github.com>
wrote:

> In modules/tensor_mechanics/src/kernels/SolidMechInertialForceTensors.C
> <https://github.com/idaholab/moose/pull/5566#discussion_r37116966>:
>
> > @@ -0,0 +1,53 @@
> > +/****************************************************************/
> > +/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
> > +/*                                                              */
> > +/*          All contents are licensed under LGPL V2.1           */
> > +/*             See LICENSE for full restrictions                */
> > +/****************************************************************/
> > +#include "SolidMechInertialForceTensors.h"
>
> Can't we just call this Kernel InertialForce? Having an abbreviated *solid
> mechanics* and a *tensors* (even though it neither is part of the solid
> mechanics module, nor does the kernel compute any tensors) seems a little
> confusing.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/idaholab/moose/pull/5566/files#r37116966>.
>